### ISSUE TITLE
Is using v1.8.0 recommended?

### ISSUE THREAD

**Issue by saitouena** [lys-lang/node-ebnf#32]
The questions are:
- Is using v1.8.0 recommended?
- Is the current latest and the most recommended version v1.7.4?

I explored the commit logs with `git log --graph --all --decorate --simplify-by-decoration`. I thought you deployed v1.8.0 by accident.

```
* commit cdd998cc5cf9b286022e041ab802763fac389db5 (origin/master, origin/HEAD, master)
| Author: Agustin Mendez <menduz@protonmail.com>
| Date:   Wed Feb 26 12:16:14 2020 -0300
| 
|     chore: remove (unused) colors package
|     
|     * Update package.json
|     
|     * remove colors from package-lock
|   
| * commit 9407fd15fac260e2bb2ba1dbe3af30f99ce91d23 (origin/add-gh-action)
| | Author: Agustin Mendez <agustin.mz.92@gmail.com>
| | Date:   Thu Jan 2 10:10:28 2020 -0300
| | 
| |     fix: add child_process dep
| | 
| * commit 33faf39ae97c3ef7d021b9c0cb8e845ff57a5cd3 (tag: v1.7.4)
| | Author: Agustin Mendez <agustin.mz.92@gmail.com>
| | Date:   Thu Jan 2 09:43:45 2020 -0300
| | 
| |     1.7.4
| | 
| * commit 91c8d1e13226799afdb5c4f47943a2cb86fd5608 (tag: v1.7.3)
| | Author: Agustin Mendez <agustin.mz.92@gmail.com>
| | Date:   Wed Jan 1 16:16:45 2020 -0300
| | 
| |     1.7.3
| | 
| * commit b32f7d9aaee18cc8b207bae893d03416e604056a (tag: v1.7.2)
|/  Author: Agustin Mendez <agustin.mz.92@gmail.com>
|   Date:   Wed Jan 1 16:13:45 2020 -0300
|   
|       1.7.2
| 
* commit 06843f318aaa9505b318623f3deb4484baf339b5 (tag: v1.7.1)
| Author: Agustin Mendez <agustin.mz.92@gmail.com>
| Date:   Sat Dec 28 12:45:04 2019 -0300
| 
|     1.7.1
|   
| * commit dcb7e4dd367ae19b0161914e044743c2e76b0a54 (HEAD, tag: v1.8.0)
|/  Author: Agustin Mendez (YUBI) <agustin.mendez@nubank.com.br>
|   Date:   Thu Dec 12 10:51:45 2019 -0300
|   
|       1.8.0
| 
* commit 1ca80911aec5f71b18f13fce76169ec11e85793b (tag: v1.7.0)
| Author: Agustin Mendez (YUBI) <agustin.mendez@nubank.com.br>
| Date:   Tue Dec 10 10:44:48 2019 -0300
| 
|     1.7.0
| 
* commit dfae70fe87dfa6a92699748b29dca2ce546204fb
  Author: Agustin Mendez <agustin.mz.92@gmail.com>
  Date:   Sun Oct 2 15:05:22 2016 -0300
      Initial commit
```

**Comment by menduz** [comment]
Hi, thanks for the heads up. I'm still using 1.7.x in `lys`.

> I thought you deployed v1.8.0 by accident.

Indeed. I need to check the other computer for the tag to push it. Otherwise I will have to do something to rollback the latest tag to 1.7.x.

Will update this soon. In the time being, I think you should use 1.7.x

**Comment by menduz** [comment]
Confirmed. `1.8.0` was a mistake. Will republish `1.9.0@latest` using `1.7.4` soon

**Comment by menduz** [comment]
Published `1.9.0`. Thank you for the heads up

**Comment by saitouena** [comment]
Thanks!
